common/src/main/java/com/wynntils/core/consumers/features/AbstractConfigurable.java

```diff
@@ -19,11 +19,6 @@ public void addConfigOptions(List<Config<?>> options) {
         configOptions.addAll(options);
     }
 
-    @Override
-    public void removeConfigOptions(List<Config<?>> options) {
-        configOptions.removeAll(options);
-    }
-
```

Comment:
Unused?

Reply:
Yes, it is unused. My presumption was that it was part of a planned design that never came about, but possibly there is a bug hiding somewhere. It's hard to search for "where *should* this have been called" in IntelliJ, so I cannot really say. :-)